socrates/lib/subscribers/subscriber.js

```diff
@@ -36,7 +36,15 @@ class Subscriber {
   }
 
   livesInGermany() {
-    return this.country() === 'DE';
+    return this.state.country && this.state.country.length > 0 && this.country() === 'DE';
```

Comment:
Ja genau, die Domänenlogik. Spätestens in index.js. Nicht hier. Wir machen hier kein OO.